### ISSUE TITLE
Export to PDF - Header /  Footer not Rendered due to pdf-lib.js Encoding Problem

### ISSUE THREAD

**Issue by madlogos** [typora/typora-issues#4485]
I tried to export a .md document to pdf. It worked, except that the header failed to render. By opening the debug mode, I found an exception in the console:

```js
frame.js:1 failed to add header / footer Error: WinAnsi cannot encode "阿"
    at sn.encodeUnicodeCodePoint (pdf-lib.min.js:15)
    at t.encodeTextAsGlyphs (pdf-lib.min.js:15)
    at t.widthOfTextAtSize (pdf-lib.min.js:15)
    at t.widthOfTextAtSize (pdf-lib.min.js:15)
    at frame.js:1
    at async FileReader.o.onloadend (frame.js:1)
```

I set header as `${title}` while the title is in Chinese. So it seems the pdf-lib.js failed to encode them when rendering the header. But the rest of the document rendered successfully, which also contains non-Ansi characters though.

Could you please look into it? Thanks.

**Comment by abnerlee** [comment]
Possible to attach or send us <hi@typora.io> the sample md file?

**Comment by madlogos** [comment]
> Possible to attach or send us [hi@typora.io](mailto:hi@typora.io) the sample md file?

Sure. Please check it out.
[测试.md](https://github.com/typora/typora-issues/files/6701535/default.md)

I tested it in Linux and MacOS, both succeeded. Only failed in Win 10.

**Comment by abnerlee** [comment]
https://github.com/Hopding/pdf-lib/issues/211

**Comment by koeiol** [comment]
windows 11 x64， Typora 1.0.2，has the same error, my file is named "赋.md".

![图片](https://user-images.githubusercontent.com/10448700/151312759-e2bfe283-d72a-4d29-ada2-029712ec6fec.png)

![图片](https://user-images.githubusercontent.com/10448700/151312802-cea4b260-629d-4767-b19c-a0d980863e67.png)

**Comment by pfugwtg** [comment]
> > Possible to attach or send us [hi@typora.io](mailto:hi@typora.io) the sample md file?
> 
> Sure. Please check it out. [测试.md](https://github.com/typora/typora-issues/files/6701535/default.md)
> 
> I tested it in Linux and MacOS, both succeeded. Only failed in Win 10.

I tested it in Debian 11 fail. What's your linux version?

**Comment by abnerlee** [comment]
fixed in 1.4.4

**Comment by YDX-2147483647** [comment]
Thanks! 🎉

However, there's a typo in [What's new 1.4 → Bug Fix → Import / Export](https://support.typora.io/What's-New-1.4/#Bug-fix) and the Change Log:

>PDF export now supports headers or footers contains non-**ASSIC** code when export.

It should be “ASCII” (American Standard Code for Information Interchange).
